flag_engine/utils/semver.py

```diff
@@ -7,15 +12,15 @@ def is_semver(value: str) -> bool:
     False
     """
 
-    return value[-7:] == ":semver"
+    return value is not None and value[-7:] == ":semver"
 
 
-def remove_semver_suffix(value: str) -> str:
+def remove_semver_suffix(value: semver.Version) -> str:
```

Comment:
Is this true? I thought we did pass a string in?

Reply:
Turns out we never did. All string values suffixed with `":semver"` are converted to `semver.Version` before this function gets called.